src/main/webapp/assets/css/feedStyle.css

```diff
@@ -235,7 +256,12 @@ select {
   }
 }
 
-#modal-error {
+#modal-map-error {
+  color: #b60000;
```

Comment:
Maybe make this a const since you reuse it.

Reply:
good idea!

Reply:
I ended up combining the two definitions since they have the exact same attributes!